packages/upswyng-web/src/components/AlertPanel.tsx

```diff
@@ -0,0 +1,78 @@
+import ExpansionPanel, {
+  ExpansionPanelProps,
+} from "@material-ui/core/ExpansionPanel";
+import Avatar from "@material-ui/core/Avatar";
+import ExpandMore from "@material-ui/icons/ExpandMore";
+import ExpansionPanelDetails from "@material-ui/core/ExpansionPanelDetails";
+import ExpansionPanelSummary from "@material-ui/core/ExpansionPanelSummary";
+import Grid from "@material-ui/core/Grid";
+import MuiMarkdown from "./MuiMarkdown";
+import React from "react";
+import { TAlertFull } from "@upswyng/upswyng-types";
+import { Theme } from "@material-ui/core/styles/createMuiTheme";
+import Tooltip from "@material-ui/core/Tooltip";
+import Typography from "@material-ui/core/Typography";
+import makeStyles from "@material-ui/styles/makeStyles";
+
+// const AlertPanelTooltip = ({children}:{})
+
+const useStyles = makeStyles((theme: Theme) => ({
+  avatar: (alert: TAlertFull) => ({
+    backgroundColor: alert.color || theme.palette.primary.main,
+    color: theme.palette.common.white,
+  }),
+  expansionPanel: (alert: TAlertFull) => ({
+    "&&, && *": {
+      opacity: !alert.detail ? 1 : undefined,
+    },
+    "&::before": {
+      backgroundColor: alert.color || theme.palette.primary.main,
+    },
+  }),
+  expansionPanelDetails: {
+    display: "block",
+  },
+  expansionPanelSummary: (alert: TAlertFull) => ({
+    cursor: !alert.detail ? "default" : undefined,
+  }),
+}));
+
+interface Props extends ExpansionPanelProps {
+  alert: TAlertFull;
```

Comment:
the users will only get a `TAlert` (`TAlertFull` is used for the admin interface)

Reply:
ah gotcha 👍 [updated](https://github.com/CodeForBoulder/upswyng/pull/309/commits/b20d6bb918c879294228f88709235728dc52acfe)